README.md

```diff
@@ -124,10 +127,41 @@ Put the `PUSH_CREDENTIAL_SID` configuration info into your application server by
     PUSH_CREDENTIAL_SID = 'CR***'
     APP_SID = 'AP***'
 
-Once you’ve done that, restart the server so it uses the new configuration info. Now it's time to test. Hit your application server's `placeCall` endpoint. This will trigger a Twilio REST API request that will make an inbound call to your mobile app. Once your app accepts the call, you should hear a congratulatory message.
+Once you’ve done that, restart the server so it uses the new configuration info. Now it's time to test. Hit your application server's `placeCall` endpoint. This will trigger a Twilio REST API request that will make an inbound call to your mobile app. You will receive a incoming call notification.
```

Comment:
Instead of `Hit your application sever`, maybe it's better to say `Use your browser to initiate an incoming call by navigating to the "localhost:3000/placeIncomingCall" route`.

Discussed with @bchen-twilio to rename the route `placeIncomingCall` for this particular functionality. And `makeOutgoingCall` for the new functionality we are adding.